rules/prefer-dom-node-dataset.js

```diff
@@ -27,6 +27,18 @@ const create = context => ({
 					optionalCall: false,
 					optionalMember: false,
 				})
+				|| isMethodCall(node, {
+					method: 'setAttribute',
+					argumentsLength: 2,
+					optionalCall: false,
+					optionalMember: true,
+				})
+				|| isMethodCall(node, {
+					methods: ['getAttribute', 'removeAttribute', 'hasAttribute'],
+					argumentsLength: 1,
+					optionalCall: false,
+					optionalMember: true,
```

Comment:
We can just remove `optionalMember: false,` above.

Reply:
u're right!
`done();`